web/store/api.ts

```diff
@@ -8,9 +8,7 @@ export function setSocketId(id: string) {
 }
 
 export const baseUrl =
-  location.port === '1234' || location.port === '3001'
-    ? `http://${location.hostname}:3001`
-    : location.origin
+  location.port === '3001' ? `http://${location.hostname}:3001` : location.origin
```

Comment:
The parcel web server runs on port 1234, so the `1234` check is still needed.
When the app is running, the front-end is served by parcel on `1234`, but express also serves the front-end on `3001`.

Reply:
do u have anything for ur 1234 port in ur nginx config? it's prolly just my nginx confict, i'll remove it from the pr tho

Reply:
no, it's for development only.